### ISSUE TITLE
[PATCH] Remove erroneous Pod link from Digest::SHA section

### ISSUE THREAD

**Issue by p5pRT** [Perl/perl5#11337]
Migrated from [rt.perl.org#90606](https://rt-archive.perl.org/perl5/Ticket/Display.html?id=90606) (status was 'resolved')

Searchable as RT90606$

**Comment by p5pRT** [comment]
### From @hinrik



---
 pod/perldelta\.pod |    2 \+\-
 1 files changed\, 1 insertions\(\+\)\, 1 deletions\(\-\)

<details><summary>Inline Patch</summary>

```diff
diff --git a/pod/perldelta.pod b/pod/perldelta.pod
index 4319436..6c2fb7e 100644
--- a/pod/perldelta.pod
+++ b/pod/perldelta.pod
@@ -1549,7 +1549,7 @@ L<Digest::SHA> has been upgraded from version 5.47 to 5.61.
 
 L<shasum> now more closely mimics L<sha1sum(1)>/L<md5sum(1)>.
 
-L<Addfile> accepts all POSIX filenames.
+addfile() now accepts all POSIX filenames.
 
 New SHA-512/224 and SHA-512/256 transforms (ref. NIST Draft FIPS 180-4
 [February 2011])
-- 
1.7.4.4
```
</details>

**Comment by p5pRT** [comment]
### From @jkeenan

On Sat May 14 23&#8203;:16&#8203;:22 2011\, hinrik wrote&#8203;:
> \-\-\-
>  pod/perldelta\.pod |    2 \+\-
>  1 files changed\, 1 insertions\(\+\)\, 1 deletions\(\-\)
> 
> diff \-\-git a/pod/perldelta\.pod b/pod/perldelta\.pod
> index 4319436\.\.6c2fb7e 100644
> \-\-\- a/pod/perldelta\.pod
> \+\+\+ b/pod/perldelta\.pod
> @&#8203;@&#8203; \-1549\,7 \+1549\,7 @&#8203;@&#8203; L\<Digest&#8203;::SHA> has been upgraded from version
> 5\.47 to 5\.61\.
> 
>  L\<shasum> now more closely mimics L\<sha1sum\(1\)>/L\<md5sum\(1\)>\.
> 
> \-L\<Addfile> accepts all POSIX filenames\.
> \+addfile\(\) now accepts all POSIX filenames\.
> 
>  New SHA\-512/224 and SHA\-512/256 transforms \(ref\. NIST Draft FIPS 180\-
> 4
>  \[February 2011\]\)

This spelling error occurred in what is now 'pod/perl5140delta\.pod'\.

$ grep \-in addfile pod/\*delta\*pod
pod/perl5140delta\.pod&#8203;:1552&#8203;:L\<Addfile> accepts all POSIX filenames\.

Is there any protocol for making corrections in older perldelta files?

Thank you very much\.
Jim Keenan

**Comment by p5pRT** [comment]
The RT System itself - Status changed from 'new' to 'open'

**Comment by p5pRT** [comment]
### From @khwilliamson

On 11/25/2011 03&#8203;:48 PM\, James E Keenan via RT wrote&#8203;:
> On Sat May 14 23&#8203;:16&#8203;:22 2011\, hinrik wrote&#8203;:
>> \-\-\-
>>   pod/perldelta\.pod |    2 \+\-
>>   1 files changed\, 1 insertions\(\+\)\, 1 deletions\(\-\)
>>
>> diff \-\-git a/pod/perldelta\.pod b/pod/perldelta\.pod
>> index 4319436\.\.6c2fb7e 100644
>> \-\-\- a/pod/perldelta\.pod
>> \+\+\+ b/pod/perldelta\.pod
>> @&#8203;@&#8203; \-1549\,7 \+1549\,7 @&#8203;@&#8203; L\<Digest&#8203;::SHA>  has been upgraded from version
>> 5\.47 to 5\.61\.
>>
>>   L\<shasum>  now more closely mimics L\<sha1sum\(1\)>/L\<md5sum\(1\)>\.
>>
>> \-L\<Addfile>  accepts all POSIX filenames\.
>> \+addfile\(\) now accepts all POSIX filenames\.
>>
>>   New SHA\-512/224 and SHA\-512/256 transforms \(ref\. NIST Draft FIPS 180\-
>> 4
>>   \[February 2011\]\)
>
> This spelling error occurred in what is now 'pod/perl5140delta\.pod'\.
>
> $ grep \-in addfile pod/\*delta\*pod
> pod/perl5140delta\.pod&#8203;:1552&#8203;:L\<Addfile>  accepts all POSIX filenames\.
>
> Is there any protocol for making corrections in older perldelta files?
>
> Thank you very much\.
> Jim Keenan
>
>
>

They are not forbidden\, but certainly not high priority\.

**Comment by p5pRT** [comment]
### From @rjbs

Thanks\, fixed in c071f8d

**Comment by p5pRT** [comment]
@rjbs - Status changed from 'open' to 'resolved'
